template_tinytemplate/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "template-tinytemplate"
+version = "2.0.0"
+authors = ["Nikolay Kim <fafhrd91@gmail.com>", "Hanif Bin Ariffin <hanif.ariffin.4326@gmail.com>"]
+workspace = ".."
+edition = "2018"
+
+[dependencies]
+env_logger = "0.7.1"
+tinytemplate = "1.1.0"
+serde = "1.0.115"
+serde_derive = "1.0.115"
+actix-http = "1.0.1"
+actix-web = "2.0.0"
+actix-rt = "1.1.1"
```

Comment:
keep it simple:

```suggestion
env_logger = "0.7"
tinytemplate = "1.1"
serde = "1"
serde_derive = "1"
actix-http = "1"
actix-web = "2"
actix-rt = "1"
```

Reply:
If you do `cargo update`, they will change this to include those MINOR.PATCH versions. Which is why I added it in. I think there _is_ an actual practical reason for this, because MINOR/PATCH could introduce a breaking change.

Reply:
Minor/patch versions should not introduce breaking changes for >1.x versions. The way cargo resolves versions makes these the same. `cargo update` does not change the Cargo.toml file, only the lockfile.